odd-platform-api/src/test/java/com/provectus/oddplatform/repository/NamespaceRepositoryIT.java

```diff
@@ -0,0 +1,40 @@
+package com.provectus.oddplatform.repository;
+
+import com.provectus.oddplatform.container.ODDPlatformDatabaseContainer;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.util.TestPropertyValues;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@Slf4j
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ContextConfiguration(initializers = {NamespaceRepositoryIT.Initializer.class})
+@RequiredArgsConstructor(onConstructor_ = {@Autowired})
```

Comment:
Having to specify autowired explicitly is not required AFAIK, just a FYI

Reply:
As weird as it is, but it didn't work. Thanks for the comment though